hasura/metadata/databases/bdjuno/tables/public_message.yaml

```
@@ -4,7 +4,14 @@ table:
 object_relationships:
 - name: transaction
   using:
-    foreign_key_constraint_on: transaction_hash
+    manual_configuration:
+      remote_table:
+        schema: public
+        name: transaction
+      insertion_order:
+      column_mapping:
+        transaction_hash: hash
+        partition_id: partition_id
```

```suggestion
      column_mapping:
        transaction_hash: hash
```
partition_id in column mapping should be dropped for the messageByAddress function to work properly